internal/version/version.go

```diff
@@ -248,3 +250,15 @@ func (i *Info) JSONString() (string, error) {
 	}
 	return string(b), nil
 }
+
+// GetLatestReleaseVersion fetches the latest github release
+// version from cri-o.
+func GetLatestReleaseVersion(token string) (string, error) {
+	ctx := context.Background()
+	client := github.NewTokenClient(ctx, token)
+	tags, _, err := client.Repositories.ListTags(ctx, "cri-o", "cri-o", nil)
+	if err != nil {
+		return "", err
+	}
+	return tags[0].GetName(), nil
```

Comment:
I think we should do something to make sure the minor version matches the branch we're currently in. Sometimes multiple releases are done in a day and this would mean we'd need to be more strategic about when those happen

Reply:
ok, with the latest push, I'm evaluating the release `Version` and checking the number of releases after that. Then picking the last release tag from the sorted list.

Reply:
But this change needs to get backported to all the major releases.